fe/fe-core/src/main/java/com/starrocks/persist/EditLog.java

```diff
@@ -637,12 +637,22 @@ public static void loadJournal(GlobalStateMgr globalStateMgr, JournalEntity jour
                 }
                 case OperationType.OP_ADD_FUNCTION: {
                     final Function function = (Function) journal.getData();
-                    GlobalStateMgr.getCurrentState().replayCreateFunction(function);
+                    String dbName = function.getFunctionName().getDb();
```

Comment:
How about add static Function replayCreateFunctionLog() and replayDropFunction(). and make these logic in the function
And call Database.replayCreateFunctionLog()?

Reply:
I did it